src/styles/utils/cards.scss

```diff
@@ -0,0 +1,82 @@
+.card {
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/222964338-ada8810c-9e75-4ab1-bfb3-28a4c5d6c8d8.png)
Block name is "card", so file should be named the same